src/HasTranslations.php

```diff
@@ -91,6 +91,18 @@ public function getTranslations($key) : array
         return json_decode($this->getAttributes()[$key] ?? '' ?: '{}', true) ?: [];
     }
 
+    public function getAllTranslations() : array
+    {
+        $translations = [];
+
+        foreach ($this->getTranslatableAttributes() as $attribute) {
+            $this->guardAgainstUntranslatableAttribute($attribute);
+            $translations[$attribute] = json_decode($this->getAttributes()[$attribute] ?? '' ?: '{}') ?: [];
```

Comment:
Couldn't we just use `getTranslations` here?

Reply:
Yes, I had done that and while debugging a few things forgot to revert it. Thanks for catching that.